frontend/src/libs/helpers/get-valid-class-names.ts

```diff
@@ -0,0 +1,6 @@
+import { type ClassValue } from 'clsx';
+import clsx from 'clsx';
```

Comment:
```suggestion
import { ClassValue, clsx } from 'clsx';
```
Can we use it in such way?

Reply:
Done!